CHANGELOG.md

```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v1.0.1](https://github.com/voxpupuli/puppet-tea/tree/v1.0.1) (2018-10-13)
+## [v2.0.0](https://github.com/voxpupuli/puppet-tea/tree/v2.0.0) (2020-11-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-tea/compare/v1.0.1...v2.0.0)
+
+**Breaking changes:**
+
+- modulesync 2.7.0 and drop puppet 4 [\#49](https://github.com/voxpupuli/puppet-tea/pull/49) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
why is this a breaking change?

Reply:
we considered dropping puppet 4 a breaking change in all of our modules

Reply:
that deserves its own entry, not just a modulesync entry